src/AutoRest.CSharp/Common/Generation/Writers/FormattableStringHelpers.cs

```diff
@@ -85,6 +85,11 @@ public static FormattableString GetIdentifiersFormattable(this IEnumerable<strin
 
         public static FormattableString GetConversionFormattable(this Parameter parameter, CSharpType toType)
         {
+            if (TypeFactory.IsList(parameter.Type) && toType.EqualsIgnoreNullable(typeof(RequestContent)))
+            {
+                return $"{typeof(RequestContent)}.{nameof(RequestContent.Create)}({parameter.Name})";
+            }
+
             var conversionMethod = GetConversionMethod(parameter.Type, toType);
```

Comment:
This new `if` is very similar with the `GetConversionMethod` method, is it possible that we combine these two methods?

Reply:
But it would be just copy exact content to `GetConversionMethod` which is not that valuable I think, because I don't want to break the switch logic in thta function.

Reply:
We want to return `RequestContent.Create("name")`, which name is `parameter.Name`, so we cannot return a string from `GetConversionMethod`

Reply:
That is why I suppose we could see some refactors here

Reply:
Synced we will keep it for now.